L/liblsl/build_tarballs.jl

```diff
@@ -29,9 +29,9 @@ if [[ ${target} == i686-linux-* || ${target} == arm-linux-* ]]; then
     export CFLAGS="-lrt -Wl,-rpath-link,/opt/${target}/${target}/lib"
 fi
 
-# Build complains that C++ 2011 support is required for Windows build
+# Enable C++ 2011 support for MinGW
 if [[ ${target} == *-w64-* ]]; then
-    export CXXFLAGS="-std=c++11"
+    export CXXFLAGS="-std=c++11 -lwinmm -lws2_32"
```

Comment:
As a note, `CXXFLAGS` are "C++ compilation flags", and linker command shouldn't go in there.  While a simple project might organize its invocations as follows:

```
$(CXX) $(CPPFLAGS) $(CXXFLAGS) -p $(OUTPUT) $(SOURCES) $(LDFLAGS)
```

A more complex project that doesn't want to try and compile all sources together into an output file in one go might instead organize as follows:
```
$(CXX) $(CPPFLAGS) $(CXXFLAGS) -o $(SOURCE1).o -c $(SOURCE1)
$(CXX) $(CPPFLAGS) $(CXXFLAGS) -o $(SOURCE2).o -c $(SOURCE2)
...
$(CXX) -o $(OUTPUT) $(SOURCE1).o $(SOURCE2).o ... $(LDFLAGS)
```

In this case, putting linker flags into `CXXFLAGS` won't work, because they won't be a part of the actual linker invocation.